pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "mini-face"
-version = "0.1.2"
+version = "0.1.4"
```

Comment:
We're not bumping versions yet - it happens only on merges from `development/*` to deployable branches